spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/connection/AbstractClientConnectionFactory.java

```diff
@@ -126,9 +139,14 @@ protected TcpConnectionSupport buildNewConnection() throws Exception {
 	 * @param socket The new socket.
 	 */
 	protected void initializeConnection(TcpConnectionSupport connection, Socket socket) {
```

Comment:
The `socket` argument isn't in use. 
I'd remove it, but this commit is intended for the backport...